meteor/app/client/menu.html

```diff
@@ -14,14 +14,17 @@
                 <li><a class="mozdef" href="/" title="MOZDEF">MOZDEF</a></li>
                 <li> <a target="_blank" href={{mozdef.kibanaURL}}>Kibana</a>
                     <ul>
-                        <li class="first"> 
+                        <li class="first">
                             {{#each kibanadashboards as item}}
                                 {{>kibanaDashboardItem}}
                             {{/each}}
                         </li>
                     </ul>
                 </li>
                 <li><a href="/alerts/">Alerts</a>
+                    <ul>
+                        <li class="first"><a href="/ipblocklist">ip blocklist</a></li>
```

Comment:
I'd think manage would be about managing mozdef itself. Lets leave it here for now, can always change it later.

Reply:
Yeah, this is what I was  wanting, a Ban Manager tab, with both a whitelist and blacklist menu item so we can manage both.
This can definitely be added later though :)